download.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ $# -ne 2 ];
+then
+    echo "download.sh backupurl backupdir"
+    exit 2
+fi
+
+set -u
+set -e
+dir=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+. ${dir}/utils
+project=${PROJECT:-$(get_project_name)}
+backupurl=$1
+backupdir=$2
+backupsrc=${BACKUPSRC:-"${backupdir}/omero_test_infra_backup.zip"}
+
+if [ ! -d ${backupdir} ];
+then
```

Comment:
I found it easier to read with the ``then`` on the line above
(this will also be consistent with one of the ``if then`` below